pymc3/backends/ndarray.py

```diff
@@ -72,11 +72,16 @@ def load_trace(directory, model=None):
     pm.Multitrace that was saved in the directory
     """
     straces = []
+    assert os.path.isdir(directory), "No such trace directory: " + directory
     for directory in glob.glob(os.path.join(directory, '*')):
         if os.path.isdir(directory):
             straces.append(SerializeNDArray(directory).load(model))
     return base.MultiTrace(straces)
 
+# Old name for load method: doesn't agree with documentation.
+def load_trace(directory, model=None):
```

Comment:
We should probably deprecate this function then? @ColCarroll what do you think?

Reply:
Hm, it looks like we import load_trace and save_trace as top-level functions. 

See https://github.com/pymc-devs/pymc3/blob/7e280b19d5dcc339c9e596656e2dba30242dc414/pymc3/__init__.py#L23 and https://github.com/pymc-devs/pymc3/blob/7e280b19d5dcc339c9e596656e2dba30242dc414/pymc3/backends/__init__.py#L120 

Do we want a top level function to be deprecated? We should probably import load instead of load_trace.

Reply:
I don't believe it's possible to import `load`, since there are multiple loads that one might replace.  I think if one was going to import load, it would be necessary to make `load` a static method on `Backend`.
One reason why `load_trace` might be there instead of load (as I remarked to @junpenglao  earlier) is that `NDarray` has a `save_trace`, which none of the other backends do.  So one can do `sample` followed by `save_trace` using the `NDarray`, but this is not possible with the other backends, which must be passed as `trace` arguments to `sample`, I believe.
Figuring out how to resolve that issue probably needs someone who has more of a big picture of PyMC3 than I do.  I am just scratching various itches I encounter when using it.